test/unit/pages/location/controller-test.js

```diff
@@ -30,4 +48,13 @@ describe('LocationController', function() {
     });
 
   });
+
+  describe('setCloseDialog', function() {
```

Comment:
This code under test is actually never used. It was left behind from previous close dialog implementation. PR https://github.com/redhat-developer-tooling/developer-platform-install/pull/487 will remove it.